### PR TITLE
Add Korean Translation

### PR DESCRIPTION
**Pull request by Tomcat-SG**
**When merged this pull request will:**
- Add Korean translation from goguryeo1

### CODE REVIEW

File: addons/circulation/stringtable.xml [hunk 1]
```diff
@@ -40,14 +44,15 @@
...
 			<Czech>Krev IV (1000ml) KS: A</Czech>
+			<Korean>A형혈액 (1000ml)</Korean>
         </Key>
         <Key ID="STR_KAT_circulation_BloodIV_B">
             <English>Blood IV (1000ml) BT: B</English>
             <German>Blut IV (1000ml) BG: B</German>
             <Polish>Krew IV (1000ml) GK: B</Polish>
             <Spanish>Sangre IV (1000ml) GS: B</Spanish>
             <Chinese>B型血液包 (1000ml)</Chinese>
-			<Czech>Krev IV (1000ml) KS: B</Czech>
```
Comment (YetheSamartaka) [inline addons/circulation/stringtable.xml:50]: This line should not be removed - probably "miss click"

**Comment by goguryeo1** [comment]
oh, i checked, updating it.
and this work is not mine, but one of my group player named TheKong.

thanks for your notice.

**Comment by Tomcat-SG** [comment]
Then tell Him thank you very much from us :)

**Comment by 1ksh1kong** [comment]
> Then tell Him thank you very much from us :)

Hello, I.m TheKong in K.G (Kimchi Guys) Arma3 Clan that made Korean translation.
When I first saw this work, I was impressed and if you don't mind, I like to use your
mod for our Clan.
Of course, We must follow Arma Public License Share-Alike.
I hope you allow us to use your mod for us so we can do advanced gameplay.
Thank you!

**Pull request merged**